Sources/Client/Model/User/User.swift

```diff
@@ -161,6 +154,28 @@ public struct User: Codable {
         }
     }
     
+    /// Safely decode user extra data and if it fail try to decode only default properties: name, avatarURL.
```

Comment:
I thought `name` and `avatarURL` were not required?

Reply:
It's by default in our docs, but it implemented as extra data.